docs/windows/deployment/overview.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Publish a .NET MAUI app for Windows"
+description: "Learn how to package and publish a Windows .NET MAUI app."
+ms.date: 03/25/2022
+---
+
+# Publish a .NET MAUI App for Windows
+
+> [!div class="op_single_selector"]
+>
+> - [Publish for Android](../../android/deployment/overview.md)
+> - [Publish for Windows](overview.md)
+
+When distributing your .NET MAUI app for Windows, you can publish the app and its dependencies to a folder for deployment to another system. You can also package the app into an MSIX package, which has numerous benefits for the users installing your app. For more information about the benefits of MSIX, see [What is MSIX?](/windows/msix/overview)
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+The current preview of .NET MAUI only allows publishing an MSIX package. You can't yet publish a Windows executable file for distribution.
+
+## Prerequisites
+
+In addition to the .NET MAUI workload, you'll need to add the latest Visual C++ build tools. Open the Visual Studio 2022 installer and add the **MSVC v143 - VS2022 C++ x64/x86 build tools (latest)** individual component. For more information on how to install an individual component, see [Modify Visual Studio workloads, components, and language packs](/visualstudio/install/modify-visual-studio?view=vs-2022&preserve-view=true).
```

Comment:
FYI: I've mentioned the need to install these tools to the team, for their consideration for including in the MAUI workload install in VS. So this may/may not eventually be removed.